styles/simple.scss

```diff
@@ -1,18 +1,106 @@
 .n-newsletter-signup--simple {
-	display: block;
+	display: flex;
+	flex-direction: row;
+	flex-shrink: 0;
 	border: 0;
+	gap: 20px;
+	padding-top: 20px;
+	background-color: oColorsByName("white-80");
+	padding-left: 24px;
+	padding-bottom: 0;
+
+	@include oGridRespondTo($from: M) {
+		align-items: start;
+		padding-left: 44px;
+		padding-bottom: 20px;
+	}
+
+	.n-newsletter-signup__card_content {
+		display: flex;
+		flex-direction: column;
+		align-items: start;
+		padding-left: 0;
+
+		@include oGridRespondTo($from: M) {
+			flex-direction: row;
+			align-items: center;
+		}
+
+	}
+
+	.n-newsletter-signup__body {
+		@include oTypographySans($scale: 0);
+		color: oColorsByName('black-80');
+		display: inline-block;
+		width: fit-content;
+
+		@include oGridRespondTo($from: M) {
+			width: 70%;
+		}
+	}
+
+	.n-newsletter-signup__top-meta {
+		top: 0;
+		right: 0;
+	}
+
+
+	.n-newsletter-signup__flex-child-left {
```

Comment:
**Question:** `flex-child-left` describes the styles/layout rather than what that element is. Is there a reason for this naming? does that element contain different things at different times?

Reply:
**Question:** I also cannot see these styles being used anywhere in the demos so can you explain where they are used. If they are used we need a demo for them and if they are not then they can be removed

Reply:
Ok I see it being used [here](https://github.com/Financial-Times/n-newsletter-signup/pull/107/files#diff-71e84ed882ed5c3832fcc269b1e324b48c71721cf9de414e8a4e6ae72e9f441dR8) so can we please add a demo for this